### PR TITLE
[IMP] im_livechat: introduce Chatbot/scriptId 

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2891949

### CODE REVIEW

File: addons/im_livechat/static/src/models/chatbot.js [hunk 1]
```diff
@@ -2,16 +2,32 @@
...
 import { attr, one } from '@mail/model/model_field';
+import { clear } from '@mail/model/model_field_command';
 
 registerModel({
     name: 'Chatbot',
     identifyingFields: ['livechatButtonViewOwner'],
+    recordMethods: {
+        /**
+         * @private
+         * @returns {integer}
```
Comment (oomsveta) [inline addons/im_livechat/static/src/models/chatbot.js:13]:
```suggestion
         * @returns {integer|FieldCommand}
```

Comment (alexkuhn) [reply addons/im_livechat/static/src/models/chatbot.js:12]: Let's see whether runbot's happy without guarding `this.data`. If not, then reintroduce the guard and make proper docstring.

**Comment by alexkuhn** [comment]
Rebased, and changes done

**Review by oomsveta — APPROVED** [review]

**Comment by alexkuhn** [comment]
Merged in grouped PR: https://github.com/odoo/odoo/pull/94722

**Pull request closed without merging**